### PR TITLE
Update companion-device-unlock.md

### PR DESCRIPTION
**Pull request by mapalko**
Updating the behavior of device registration for companion devices in the next major release of Windows and adding information around policy management.

### CODE REVIEW

**Comment by garycentric** [comment]
@mijacobs  : The PR **passed** the acceptance checklist. Please evaluate further and approve/complete the PR if appropriate. Thanks!

**Comment by mijacobs** [comment]
Adding @hickeys, who owns this article.

**Comment by mapalko** [comment]
@hickeys can you review this open change? I am the current PG area owner.

**Comment by PRMerger10** [comment]
@mapalko : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-apps-src/security/companion-device-unlock.md [hunk 1]
```diff
@@ -68,6 +71,8 @@ Any number of these user signals can be combined into one. User presence and int
 
 ### Registration and future communication between a PC and Windows Hello companion devices
 
+> In the next major release of Windows following 1809, there is a behavioral change to companion device registration. For users that currently have provisioned and are using a companion device and upgrade to this release, there will be no changes. For users that are not using the companion device framework and upgrade, policy must be enabled before setting up a companion device with Windows Hello.
```
Comment (hickeys) [inline windows-apps-src/security/companion-device-unlock.md:74]: Was this intended to be a warning like the previous change? If so, it's missing `[!WARNING]

**Review by hickeys — CHANGES_REQUESTED** [review]
Looks good except for my one question. Happy to approve this after it's resolved.

File: windows-apps-src/security/companion-device-unlock.md (hunk 1 above)
Comment (mapalko) [reply windows-apps-src/security/companion-device-unlock.md:74]: It's meant to be informational. I think maring this as a Warning would be acceptable.

**Comment by mapalko** [comment]
@hickeys please review my response to your question.

**Comment by mapalko** [comment]
@hickeys any updates on this PR?

**Pull request closed without merging**